admin/package.json

```
@@ -34,7 +34,8 @@
     "lint:translations": "eslint --ext .json --format node_modules/eslint-plugin-i18n-json/formatter.js translations",
     "lint:translations:fix": "npm run lint:translations -- --fix",
     "preinstall": "npx check-engine",
-    "start": "ember serve --proxy http://localhost:3000",
+    "dev": "ember serve --proxy http://localhost:3000",
```

Micro détail, mais j'aurais plutôt fait l'inverse (dire que le dev pointe sur le start standard) :
```suggestion
    "dev": "npm run start",
    "start": "ember serve --proxy http://localhost:3000",
```